### DISCUSSION TITLE
Emulate Minimize / maximize window Offscreen

### DISCUSSION THREAD

**Discussion by bbday** [cefsharp/CefSharp#3687] [General]
Is possible add on Offscreen version command to send minimize / maximize event like wpf version?
I have a web page that detect when you minimize or maximize window and work good on wpf version but not work with offscreen version cause you can't emulate this command.

**Comment by amaitland** [comment]
Please use https://github.com/cefsharp/CefSharp/discussions rather than continuing to open feature requests for what are actually questions. 

Minimize/Maximize simply change the window size, you can easily do this already with OffScreen version.

**Reply by bbday** [reply]
Thanks for you reply.
If you mean browser.Size = new Size(0, 0) not work, i tested it (it is also wrong cause will change screen.width / height).
I need same event of wpf version when minimize window that work great.

**Reply by amaitland** [reply]
http://cefsharp.github.io/api/91.1.x/html/M_CefSharp_IBrowserHost_WasHidden.htm
http://cefsharp.github.io/api/91.1.x/html/M_CefSharp_WebBrowserExtensions_GetBrowserHost.htm

The WPF version uses the same internally. You can refer to the WPF source as a reference.
